### PR TITLE
update doc for development topic

### PR DESCRIPTION
**Pull request by vkagamlyk**
*Issue #, if available:*
https://bitquill.atlassian.net/browse/AN-988

*Description of changes:*
doc for development topic

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/src/dev/developer/development-environment.asciidoc [hunk 1]
```diff
@@ -248,6 +248,23 @@ can be used to help run tests.
 
 See the <<release-environment,Release Environment>> section for more information on release manager configurations.
 
+[[go-environment]]
+=== Go Environment
+
+The build optionally requires link:https://go.dev/dl/[Go] (>=1.7) to work with the `gremlin-go` module. If Go is not installed, TinkerPop will still build with Maven, but Go projects will be skipped.
```
Comment (lyndonbauto) [inline docs/src/dev/developer/development-environment.asciidoc]: Version > 1.17.

File: docs/src/dev/developer/release.asciidoc [hunk 2]
```diff
@@ -242,12 +242,15 @@ for generating javadoc.
...
 .. `mvn deploy -pl gremlint -DskipTests -Dnpm`
+.. `git tag gremlin-go/v3.5.4`
+.. `git push --tags`
 . Review the GLV releases
 .. link:https://pypi.org/project/gremlinpython/[gremlin-python - PyPi]
 .. link:https://www.nuget.org/packages/Gremlin.Net/[Gremlin.Net - nuget]
 .. link:https://www.nuget.org/packages/Gremlin.Net.Template/[Gremlin.Net.Template - nuget]
 .. link:https://www.npmjs.com/package/gremlin[gremlin - npm]
 .. link:https://www.npmjs.com/package/gremlint[gremlint - npm]
+.. link:https://pkg.go.dev/github.com/apache/tinkerpop/gremlin-go/v3/driver/[gremlin-go - go packages]
```
Comment (lyndonbauto) [inline docs/src/dev/developer/release.asciidoc]: Add 'v3' to hyperlink

File: docs/src/dev/developer/development-environment.asciidoc (hunk 1 above)
Comment (vkagamlyk) [reply docs/src/dev/developer/development-environment.asciidoc]: fixed in https://github.com/lyndonbauto/tinkerpop/pull/83/commits/1001ef64e2ae1df6ee8813ea5dbeac2f4bff0a19

File: docs/src/dev/developer/release.asciidoc (hunk 2 above)
Comment (vkagamlyk) [reply docs/src/dev/developer/release.asciidoc]: fixed in https://github.com/lyndonbauto/tinkerpop/pull/83/commits/1001ef64e2ae1df6ee8813ea5dbeac2f4bff0a19

File: docs/src/dev/developer/development-environment.asciidoc [hunk 3]
```diff
@@ -248,6 +248,23 @@ can be used to help run tests.
...
+
+`gremlin-go` can be built the command line with:
+
+[source,text]
+go build
+
+Docker allows you to test the driver without installing any dependencies. The following command can be used to run docker:
+
+[source,text]
+docker-compose up --exit-code-from integration-tests
```
Comment (lyndonbauto) [inline docs/src/dev/developer/development-environment.asciidoc]:
```suggestion
docker-compose up --exit-code-from gremlin-go-integration-tests
```

**Review by lyndonbauto — APPROVED** [review]

**Pull request merged**